template/src/components/ui/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, IconButton, styled } from "@mui/material";
+import * as React from "react";
+import { CustomDialog } from "./CustomDialog";
+import { Icon } from "./Icon";
+
+const HeaderContainer = styled("div")({
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+});
+
+const MessageContainer = styled("div")({
+    textAlign: "left",
```

Comment:
Isn't this the default value?

Reply:
[remove messagecontainer](https://github.com/allaboutapps/cra-template-aaa-typescript/pull/23/commits/ab63ab684389f130425c33896cfdc3e1fa616b32)